docs/designers-developers/developers/tutorials/javascript/troubleshooting.md

```diff
@@ -0,0 +1,35 @@
+## Troubleshooting
+
+It's not working, what do I do?
+
+### Console Log
+
+The console log is a JavaScript developer's best friend, it is a good practice to work with it open. If there are any errors this is where they will show up. See Mozilla's [JavaScript console](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/What_are_browser_developer_tools#The_JavaScript_console) documentation for more.
+
+Check the JavaScript console for any errors. Here is an example, which shows a syntax error on line 6.
+
+![console error](../../../../../docs/designers-developers/developers/tutorials/javascript/console-error.png)
+
+
+### Confirm all dependencies are loaded
+
+The console log will show an error if a dependency your JavaScript code uses has not been declared and loaded in the browser. In the example, if `myguten.js` script is enqueued without declaring the `wp-block` dependency, the console log will show:
+
+<img src="../../../../../docs/designers-developers/developers/tutorials/javascript/error-blocks-undefined.png" width=448 title="error wp.blocks is undefined"/>
```

Comment:
Can we use the markdown image format here?

Reply:
I tried but the image comes in too wide since it was created on a @2x screen.

Also tried using markdown formatting here: https://gist.github.com/uupaa/f77d2bcf4dc7a294d109 but they didn't work either. 

Open to suggestions, I can size it down but then won't look good on high dpi screens.